usecaseImpl/home/src/commonTest/kotlin/tech/takahana/iconwallpaper/usecase/home/HomeSelectPatternUseCaseImplTest.kt

```diff
@@ -0,0 +1,90 @@
+package tech.takahana.iconwallpaper.usecase.home
+
+import io.mockk.MockKAnnotations
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.mockk
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.runTest
+import tech.takahana.iconwallpaper.repository.asset.SelectImageAssetRepository
+import tech.takahana.iconwallpaper.repository.asset.SelectPatternTypeRepository
+import tech.takahana.iconwallpaper.shared.assets.LocalImageAsset
+import tech.takahana.iconwallpaper.shared.domain.domainobject.AssetId
+import tech.takahana.iconwallpaper.shared.domain.domainobject.AssetName
+import tech.takahana.iconwallpaper.shared.domain.domainobject.PatternType
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+@ExperimentalCoroutinesApi
+class HomeSelectPatternUseCaseImplTest {
+
+    @MockK
+    lateinit var mockSelectPatternTypeRepository: SelectPatternTypeRepository
+
+    @MockK
+    lateinit var mockSelectImageAssetRepository: SelectImageAssetRepository
+
+    @BeforeTest
+    fun setUp() {
+        MockKAnnotations.init(this)
+    }
+
+    @Test
+    fun selectPattern() = runTest {
+        val mockPatternType: PatternType = mockk()
```

Comment:
メソッドなどのモックをしてなさそうなので、モックではなく実際のPatternTypeを使っていいかも👀
```suggestion
        val mockPatternType: PatternType = PatternType.LARGE
```